js/ui/grid_core/ui.grid_core.adaptivity.js

```diff
@@ -79,7 +80,7 @@ var AdaptiveColumnsController = modules.ViewController.inherit({
             var templateOptions = extend({}, cellOptions, { value: cellValue, text: cellText, column: column });
             that._rowsView.renderTemplate($container, column.cellTemplate, templateOptions, !!$container.closest(document).length);
         } else {
-            var container = $container.get(0);
+            container = $container.get(0);
```

Comment:
It looks like that we do not need this row at all

Reply:
We need this row. container variable can be jQuery object and jQuery doesn't have textContent property...

Reply:
ok